lib/locreator.js

```diff
@@ -75,6 +75,12 @@ Locreator.prototype.addStarMethods = function locreator(locatorId, _locator, par
   locatorObject[locatorId + 'OptionValue'] = function (optionValue) {
     return drivex.selectByOptionValue(locator, optionValue, parentWebElement);
   };
+  locatorObject[locatorId + 'ValidateText'] = function (ValidateText) {
+    return drivex.validateText(locator, parentWebElement,ValidateText) ;
+  };
+  locatorObject[locatorId + 'ValidateAttributeValue'] = function (attribute,validateText) {
```

Comment:
What about calling the method "AttributeEquals" or "AttrEquals"? Trying to shorten the name. @nikulkarni what do you think?